src/main/resources/config/liquibase/akip-master.xml

```diff
@@ -38,5 +38,7 @@
 
     <include file="config/liquibase/changelog/00000000002001_added_entity_AkipEmailConnectorConfig.xml" relativeToChangelogFile="false"/>
 
+    <include file="config/liquibase/changelog/20230222000001_added_entity_DecisionDefinition.xml" relativeToChangelogFile="false"/>
```

Comment:
@CassioJr, I forgot to mention: in the `akip-master.xml` file, the changesets are not ordered by date. Instead, we use a sequence order. So, to keep the pattern, the new changesets should start with `000..14`..`000...15`